zipkin-web/src/main/resources/app/js/component_ui/trace.js

```diff
@@ -287,7 +287,9 @@ define(
             $.each(self.parents[id], function(i, pId) { self.spans[pId].openChildren += 1; });
           });
           $.each(self.spansByService, function(svc, spans) {
-            $.each(spans, function(i, $span) { $span.inFilters += 1; });
+            $.each(self.getSpansByService(svc), function(i, $span) {
```

Comment:
What the code is trying to do is, `getAllSpans`, which isn't implemented, so it loops through all the service names, and then gets all spans for every service name.